resource/storage.go

```diff
@@ -155,7 +155,7 @@ func (s storageWrapper) MultiGet(ctx context.Context, ids []interface{}) (items
 }
 
 // Find tries to use storer MultiGet with some pattern or Find otherwise
-func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage int) (list *ItemList, err error) {
+func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage, offset int) (list *ItemList, err error) {
```

Comment:
well. yes I agree. for the storage backend you could do the translation in [rest/method_get.go](https://github.com/rs/rest-layer/pull/62/commits/ca55f79fc1e83a989ea5912c470c358d3ad02cf6#diff-a90cd58ba7082f803eb1ab4b75bd612a) and avoid one more func param.